connect/transforms/src/main/java/org/apache/kafka/connect/transforms/Cast.java

```diff
@@ -188,6 +196,7 @@ private Schema getOrBuildSchema(Schema valueSchema) {
         }
 
         if (valueSchema.isOptional())
+            log.trace("Cast SMT: ValueSchema is optional");
```

Comment:
This probably doesn't add value without more information about the name of the schema.